core/src/test/java/org/apache/accumulo/core/file/rfile/RFileTest.java

```diff
@@ -2271,4 +2273,40 @@ public void testRootTabletEncryption() throws Exception {
 
     conf = null;
   }
+
+  @Rule
+  public ExpectedException exception = ExpectedException.none();
+
+  @Test
+  public void testKeyTooLarge() throws Exception {
+    // If we don't have enough memory (6G) skip these very memory-intensive unit tests
+    Assume.assumeTrue(Runtime.getRuntime().maxMemory() >= 5726797824L);
```

Comment:
It's not clear where these numeric literals come from. Perhaps it can be put in a variable, and their computation shown, so it's clear where they come from? If they are arbitrary, then a comment explaining that (eg. "arbitrary size in the range of ... ") would be helpful.